docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -33,11 +33,117 @@ Understanding GBD's Disease and Health Categories
 GBD Definitions
 ---------------
 
-- Include the GBD definitions for each of these different categories 
+These definitions and others can be found on the `glossary Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=INTRANET&title=IHME+Glossary>`_. 
 
-Understanding the GBD Hierarchy
--------------------------------
+.. list-table:: GBD definitions of terms 
+  :widths: 15 15 15
+  :header-rows: 1
 
-- Showing what is a direct subset, what overlaps, what interacts but is independent 
-- Include some information on modeling implications as a result of this 
+  * - Term 
+    - Definition
+    - Notes
+  * - Cause
+    - A cause of death or disability. Includes injuries, diseases, and conditions. 
+    - See cause list for more details 
+  * - Cause List 
+    - The list of all causes in GBD. It is a mutually exclusive and collectively exhaustive list of hierarchical categories that does not ignore any cause of death. All death and disability within GBD can be found within the cause list. 
+    - 
+  * - Health State 
+    - The current state of health associated with a given cause and sequela.
+    - 
+  * - Impairment
+    - The symptoms of a disease, such as vision loss as a result of diabetes. Clinically the same as sequelae. 
+    - GBD calculates these very differently than sequelae despite them being clinically the same. 
+  * - Sequela or Sequelae 
+    - The medical conditions that can occur among people who contract a disease or suffer an injury. In other words, the negative health effects of a cause that are associated with certain health states. For example, chronic kidney disease can be a sequela of diabetes, neck pain can be a sequela of whiplash, and foot ulcers is a sequela of diabetes.
```

Comment:
I think that GBD tries to make these mutually exclusive and collectively exhaustive within a given cause -- we should check on that and note it here if that is true.

Reply:
It may also be worth noting here that there are asymptomatic sequelae.